src/keys/keypair.rs

```diff
@@ -27,8 +27,6 @@ use threshold_crypto::{self, serde_impl::SerdeSecret};
 pub enum Keypair {
     /// Ed25519 keypair.
     Ed25519(ed25519_dalek::Keypair),
-    /// BLS keypair.
-    Bls(BlsKeypair),
```

Comment:
Not sure why we are removing this?
(On a side note, we use BLS keypairs a lot at sn_client(esp for testing), removing this would induce a big refactor at client I think)